### PR TITLE
docs: clarify instructions, OS indepedence

### PR DESCRIPTION
**Pull request by zalberico**
This is an iteration/polish improvement on my previously merged PR: https://github.com/urbit/urbit.org/pull/468

Details listed below, but core overview is that I went through and brought the planet boot instructions in line with the new directory structure used in setting up comets (to make binary upgrades easier for users). I also changed the example to just show users commands for both macOS and Linux.

I think this simplifies things for users by making things more explicit - I think showing the key deletion command (while trivial) will also make it less likely for users to glance over that comment in the paragraph and miss the step (in my experience people tend to notice code blocks more than they do sentences in text).

* Show explicit commands for both macOS and Linux where appropriate
* Bring planet boot instruction directory structure into alignment with instructions for comets
* Show explicit command for removing keyfile (harder to miss than note in paragraph)
* In binary upgrade example, remove old binary before untarring new one (prevents name collision)
* In binary upgrade example just show users explicit commands for macOS and Linux.
* Fix missing breaches tag link

### CODE REVIEW

**Comment by zalberico** [comment]
@baudtack, @matildepark, @botter-nidnul for SA - thanks!

File: content/using/install.md
Comment (matildepark) [inline content/using/install.md]: `Ctrl+D` is actually clearest. But be consistent with the `Ctrl-x` right above it...

Comment (zalberico) [reply content/using/install.md]: I'll make the change

File: content/using/install.md
Comment (matildepark) [inline content/using/install.md]: A little verbose. We're about to demonstrate that it's the same ship on line 327

Comment (zalberico) [reply content/using/install.md]: I'm worried people who have previously put their ship in their binary directory might delete it if they don't take it out first.

So it's a little verbose, but I thought explicitly stating ships can be reused and to not delete your old one would be good.

**Comment by matildepark** [comment]
Screenshot 2020-09-06 at 1 10 51 AM
Screenshot 2020-09-06 at 1 10 55 AM

inconsistent

**Comment by zalberico** [comment]
I think this is good to merge

**Comment by matildepark** [comment]
lol, but the first style was better tho

**Comment by zalberico** [comment]
I think the first style takes up a lot more vertical screen space without any real clarity benefit.

**Comment by zalberico** [comment]
Unless you have really strong objections to this, I'd vote to merge (I think it's a nit/preference?).

Obviously if this will block merge I'll change it.

After this merges I think the install docs will be in a really good place (I don't have any more PRs planned for now).

Next up I'm going to take a look at some of the star specific stuff and might have some new docs then.

**Comment by matildepark** [comment]
I disagree on what looks better but would take a trade off where you mark the OS lines as comments, so they aren’t given bash syntax highlighting

**Review by matildepark — APPROVED** [review]

**Pull request merged**
